test/powershell/Language/Parser/LineContinuance.Tests.ps1

```diff
@@ -0,0 +1,247 @@
+            $script = @'
+'Hello' + ` # You can place comments after whitespace following backticks
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a single backtick followed by a comment line and then the continued line' {
+            $script = @'
+'Hello' + `
+# You can place comments in the middle of a continued line
+    ' world'
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+    }
+
+    Context 'Lines ending with a backtick that do not parse' {
+        It 'Lines ending with a single backtick followed immediately by a comment' {
+            $script = @'
+'Hello' + `# This is not a valid comment
+    ' world'
+'@
+            $err = { ExecuteCommand $script } | Should -Throw -ErrorId 'ParseException' -PassThru
+            $err.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -BeExactly 'ExpectedValueExpression'
+        }
+
+        It 'Lines ending with two backticks' {
+            $script = @'
+'Hello' + ``
+    ' world'
+'@
+            $err = { ExecuteCommand $script } | Should -Throw -ErrorId 'ParseException' -PassThru
+            $err.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -BeExactly 'ExpectedValueExpression'
+        }
+    }
+
+    Context 'Lines ending with a pipe that parse and execute without error' {
+        It 'Lines ending with a pipe' {
+            $script = @'
+'Hello' |
+    ForEach-Object {"$_ world"}
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a pipe followed by whitespace' {
+            $script = @'
+# The next line ends with trailing whitespace
+'Hello' |     
+    ForEach-Object {"$_ world"}
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a pipe followed by a comment' {
+            $script = @'
+'Hello' |# You can place comments after whitespace following pipes
+    ForEach-Object {"$_ world"}
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+
+        It 'Lines ending with a pipe followed by a comment line and then the continued command' {
+            $script = @'
+'Hello' |
+# You can place comments in the middle of a continued pipeline
+    ForEach-Object {"$_ world"}
+'@
+            ExecuteCommand $script | Should -Be 'Hello world'
+        }
+    }
+
+    Context 'Lines ending with a pipe that do not parse' {
+        It 'Lines ending with a single pipe followed by an empty line' {
+            $script = @'
+'Hello' |
+
+'@
+            $err = { ExecuteCommand $script } | Should -Throw -ErrorId 'ParseException' -PassThru
+            $err.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -BeExactly 'EmptyPipeElement'
+        }
+
+        It 'Lines ending with two pipes' {
+            $script = @'
+'Hello' ||
```

Comment:
Should we check:
```powershell
           $script = @' 
'Hello' |
        | 
'@

Reply:
Yes, that is a better test. The other test is necessary, but it's not a line continuance test, so I moved it to Parsing.Tests.ps1.